src/index.js

```diff
@@ -0,0 +1,30 @@
+import './style.css';
+import {
+  getStorageData, clearCompletedList, clearCheckBox, addTodo, saveEdit, displayToDo, getIsEditing,
+} from './module/addRemove.js';
+
+window.onload = () => {
+  getStorageData();
+  displayToDo();
+};
+
+const refreshBtn = document.querySelector('.refresh-btn');
+refreshBtn.addEventListener('click', () => {
+  clearCheckBox();
+  window.location.reload();
+});
+
+const descriptions = document.querySelector('.inputTask');
+descriptions.addEventListener('keyup', (event) => {
+  if (event.keyCode === 13) {
```

Comment:
- [ ] I suggest that you replaced `event.keyCode` with `event.key === 'Enter'`
      to handle the Enter key press event in a more modern and reliable way. :tada: :tada:

Reply:
Thank you for your valuable suggestion @Ol-create :100: